src/coordinator/generated/proto/rpc/query.proto

```diff
@@ -62,8 +62,36 @@ message FetchResult {
 	repeated Series series = 1;
 }
 
+message Segment {
+	bytes head = 1;
+	bytes tail = 2;
+}
+
+message Segments {
+	Segment merged = 1;
+	repeated Segment unmerged = 2;
+}
+
+message CompressedValuesReplica {
+	repeated Segments segments = 1;
+}
+
+message CompressedTag {
+	bytes name = 1;
+	bytes value = 2;
+}
+
+message CompressedDatapoints {
```

Comment:
hm weird that this has a namespace and values but no id? 

could you break into: `CompressedDatapoints` and `CompressedSeries` maybe?

Reply:
Might leave it for now since I want to move most of the CompressedSeries up into in the next PR when I flip the remote fetch path to use this new encoding, i.e. I'll remove `CompressedSeries` if possible and change `Series` to something like:

```
message Series {
	bytes id = 1;
	bytes namespace = 2;
	int64 startTime = 3;
	int64 endTime = 4;
	repeated CompressedTag tags = 4;
	repeated CompressedValuesReplica replicas = 5;
}
```